Guide/authentication.markdown

```diff
@@ -259,6 +260,22 @@ You can also access the user using `currentUser` inside your views:
 |]
 ```
 
+## Performing actions after login
```

Comment:
The function is called `beforeLogin` and the headline says `after login`, this might be a bit confusing :) Maybe we can change the headline to this:

```suggestion
## Performing actions on login
```